apps/elasticsearch/templates/services/app/configs/set-index-template.sh.j2

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+MAX_RETRY=20
+RETRY_DELAY=5
+
```

Comment:
I would add:

```bash
set -eo pipefail
```

Reply:
I didn't because : 

- The first part of the script (the curl loop) can fail . There is a `set -e to` ensure that nothing fails after line 14.
- I didn't add `-o pipefail` because there is no usage of the pipe operator in this script.

Reply:
> The first part of the script (the curl loop) can fail . There is a set -e to ensure that nothing fails after line 14.

:thinking: I hesitated, I thought it would be catched by the conditional block.

> I didn't add -o pipefail because there is no usage of the pipe operator in this script.

For now, but what if we add one in a future version? You can be 99% sure that we will forget to add it, when required. You can consider this more like a convention.

Reply:
You're right, it's caught by the conditional block !

Go for `set -eo pipefail` :tada: